src/styles/global.css

```diff
@@ -36,6 +36,13 @@
     @apply text-sm sm:text-base;
   }
 
+  a {
+    @apply underline;
+  }
+  a:hover {
+    @apply no-underline;
+  }
```

Comment:
We have a couple link related classes in this file. So maybe we can consolidate it / make sure they are not overriding each other.

Reply:
ah didnt see that, thx